spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnSelector.scala

```diff
@@ -28,7 +28,10 @@ case class SomeColumns(columns: ColumnRef*) extends ColumnSelector {
   }.toMap
 
   override def selectFrom(table: TableDef): IndexedSeq[ColumnRef] = {
-    val missing = table.missingColumns(columns).filterNot(_ == RowCountRef)
+    val missing = table.missingColumns(columns).filterNot {
+      case _: FunctionCallRef => true
```

Comment:
I also think it would be cool if we checked the columns that are accessed by the functions here, for that we don't actually need to know the Function list